Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -54,6 +54,15 @@ extension Issue {
     return issue.record(configuration: configuration)
   }
 
+  /// Record this issue by wrapping it in an ``Event`` and passing it to the
```

Comment:
I don't think we really discuss events or event handlers in our API documentation because they're not otherwise part of our API surface. How about simply:

```swift
/// Record this issue.
```